room/room-paging-guava/src/main/androidx/room/androidx-room-room-paging-guava-documentation.md

```diff
@@ -0,0 +1,7 @@
+# Module root
```

Comment:
Empty package docs? Maybe just remove this if we're gonna leave it blank and we can add it back later if we actually have something to write here.

Reply:
Got it -- removed. I wasn't sure what to do with it.